RELEASES.md

```diff
@@ -1,3 +1,123 @@
+Version 1.50.0 (2021-02-11)
+============================
+
+Language
+-----------------------
+- [You can now use `const` values for `x` in `[x; N]` array expressions.][79270]
```

Comment:
Note that *"`const` values"* is not exactly right, depending on interpretation. `[Vec::<i32>::new(); 10]` is not accepted. It only works for actual constants, not values/expressions that are (or can be) const: e.g. `[EMPTY; 10]` with `const EMPTY: Vec<i32> = Vec::new();`.